proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,239 @@
+# dotnet nuget trust command
+
+- Author Name [Kat Marchan (@zkat)](https://github.com/zkat)
+- Start Date 2021-03-04
+- GitHub Issue: https://github.com/NuGet/Home/issues/10634
+- GitHub PR: https://github.com/NuGet/Home/pull/10628
+
+## Summary
+
+This specifies the syntax and behavior for the `dotnet nuget trust` command.
+
+## Motivation
+
+This is part of a larger effort to reach full parity between `nuget` and
+`dotnet` for NuGet-related operations.
+
+## Explanation
+
+### Functional explanation
+
+#### `> dotnet nuget trust`
+
+```
+Provides the ability to manage the list of trusted signers.
+
+USAGE:
+    dotnet nuget trust [COMMAND] [OPTIONS]
+
+COMMANDS:
+    list
+    author <name> <package>...
+    repository <name> <package>...
+    source <name> [source-url]
+    certificate <name> <fingerprint>
+    remove <name>
+    sync <name>
+```
+
+#### `> dotnet nuget trust list`
+
+```
+Lists all the trusted signers in the configuration. This option will include all the certificates (with fingerprint and fingerprint algorithm) each signer has. If a certificate has a preceding [U], it means that certificate entry has allowUntrustedRoot set as true.
+
+USAGE:
+    dotnet nuget trust list [OPTIONS]
+
+EXAMPLE:
+    $ dotnet nuget trust list
+    Registered trusted signers:
+
+    1.   nuget.org [repository]
+        Service Index: https://api.nuget.org/v3/index.json
+        Owners: microsoft, aspnet, nuget
+        Certificate fingerprint(s):
```

Comment:
It looks like the name has an extra space in the output. Is that intention?

Reply:
nope :)